build.sh

```diff
@@ -195,6 +208,15 @@ check_dependencies() {
 }
 
 show_command=0
+os=""
+dist=""
+version=""
+edition=""
+auto_continue=false
+# auto_retry=false
+# Script options.
+
+
 # Script options.
 while (("$#")); do
     case "$1" in
```

Comment:
```suggestion
auto_continue=false

# Script options.
while (("$#")); do
    case "$1" in
```

Reply:
https://github.com/ClemTheStudent/packer-examples-for-vsphere/commit/aa5102d23bd5d4ac112a03545c21dfce5b2d511a